core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -76,6 +77,13 @@ oppia.factory('PlayerTranscriptService', ['$log', function($log) {
       });
       numAnswersSubmitted = 0;
     },
+    addPreviousCard: function() {
+      if (transcript.length === 1) {
```

Comment:
Can this case actually happen in practice? If not, throw an error.

Reply:
Yeah, no, it shouldn't added the throw Error.